src/Sdk/HistoryApiInterface.php

```diff
@@ -66,10 +68,10 @@ public function findAllOperations(ChainObject $accountId, array $assets = [], Ch
      * NOTICE:
      * Transaction object with id in form '1.7.X' can be fetched from AccountApi->getAccountHistory() method.
```

Comment:
I think it is called history object to be consistent.

Reply:
Renamed to `History object...`